internal/bminventory/inventory.go

```diff
@@ -4531,6 +4532,14 @@ func (b *bareMetalInventory) getOpenshiftVersionFromOCP(log logrus.FieldLogger)
 	return k8sclient.GetClusterVersion(clusterVersion)
 }
 
+func (b bareMetalInventory) DeregisterInactiveClusters() {
```

Comment:
I don't think that it should be in the inventory.

Reply:
where do you recommend locating this?

Reply:
Create a garbage collection package.